scripts/monorepo/compass-deps-list.js

```diff
@@ -0,0 +1,400 @@
+  },
+  {
+    name: '@mongodb-js/compass-instance',
+    description: 'compass instance plugin',
+    repository: 'https://github.com/mongodb-js/compass-instance.git'
+  },
+  {
+    name: '@mongodb-js/compass-license',
+    repository: 'https://github.com/mongodb-js/compass-license.git'
+  },
+  {
+    name: '@mongodb-js/compass-loading',
+    description: 'Compass Loading Screen',
+    repository: 'https://github.com/mongodb-js/compass-loading.git'
+  },
+  {
+    name: '@mongodb-js/compass-metrics',
+    description: 'Compass Metrics Plugin',
+    repository: 'https://github.com/mongodb-js/compass-metrics.git'
+  },
+  {
+    name: '@mongodb-js/compass-plugin-info',
+    description: 'Compass Plugin Information Plugin',
+    repository: 'https://github.com/mongodb-js/compass-plugin-info.git'
+  },
+  {
+    name: '@mongodb-js/compass-query-bar',
+    description:
+      'Renders a component for executing MongoDB queries through a GUI.',
+    repository: 'https://github.com/mongodb-js/compass-query-bar.git'
+  },
+  {
+    name: '@mongodb-js/compass-query-history',
+    description: 'The query history sidebar.',
+    repository: 'https://github.com/mongodb-js/compass-query-history.git'
+  },
+  {
+    name: '@mongodb-js/compass-schema',
+    description: 'Compass Schema Tab Plugin',
+    repository: 'https://github.com/mongodb-js/compass-schema.git'
+  },
+  {
+    name: '@mongodb-js/compass-schema-validation',
+    description: 'Compass plugin for document JSON schema validation',
+    repository: 'https://github.com/mongodb-js/compass-schema-validation.git'
+  },
+  {
+    name: '@mongodb-js/compass-server-version',
+    description: 'Compass Server Version',
+    repository: 'https://github.com/mongodb-js/compass-server-version.git'
+  },
+  {
+    name: '@mongodb-js/compass-serverstats',
+    description: 'Compass Real Time Server Stats Component.',
+    repository: 'https://github.com/mongodb-js/compass-serverstats.git'
+  },
+  {
+    name: '@mongodb-js/compass-shell',
+    description: 'Compass Shell Plugin',
+    repository: 'https://github.com/mongodb-js/mongosh.git',
+    // Because we don't want to get the whole history of mongosh and there is no
+    // way to extract it in a better way
+    importStrategy: 'copy'
+  },
+  {
+    name: '@mongodb-js/compass-sidebar',
+    description: 'Sidebar external plugin',
+    repository: 'https://github.com/mongodb-js/compass-sidebar.git'
+  },
+  {
+    name: '@mongodb-js/compass-ssh-tunnel-status',
+    description: 'Compass SSH Tunnel Status',
+    repository: 'https://github.com/mongodb-js/compass-ssh-tunnel-status.git'
+  },
+  {
+    name: '@mongodb-js/compass-status',
+    description: 'Compass Status Plugin',
+    repository: 'https://github.com/mongodb-js/compass-status.git'
+  },
+  {
+    name: '@mongodb-js/dl-center',
+    description: 'mongodb download center internal tools',
+    repository: 'https://github.com/mongodb-js/dl-center.git'
+  },
```

Comment:
We do use this in mongosh, does it make sense to keep that somewhat separate?

Reply:
Totes, let's remove it!

Reply:
Removed in fd579d4